DependencyInjection/Configuration.php

```diff
@@ -87,6 +87,7 @@ public function getConfigTreeBuilder()
                 ->canBeEnabled()
                 ->children()
                     ->scalarNode('ip')->defaultNull()->end()
+                    ->booleanNode('use_faker')->defaultFalse()->end()
```

Comment:
Should we throw if `use_faker` is true and faker is not installed?

Reply:
faker was added to composer.json (require-dev) so, maybe I am wrong, but faker will always be installed, isn't it?

Reply:
No, dev requirements doesn't ensure that it will be in production unless required explicitly in dependencies. Also it must be optional, otherwise others will get unnecessary dependency if they don't want to use faker.